### PR TITLE
remove no v-html enforcing rule

### PR DESCRIPTION
**Pull request by rowbi-wan**
Rather than overriding this manually in each project Ive created a rule for it.

### CODE REVIEW

File: packages/eslint-config/rules/vue/no-v-html/rule.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+module.exports = {
+    rules: {
+        'vue/no-v-html': 0,
```
Comment (samturrell) [inline packages/eslint-config/rules/vue/no-v-html/rule.js]: I actually don't know what this will generate as a severity level in the docs...

Comment (rowbi-wan) [reply packages/eslint-config/rules/vue/no-v-html/rule.js]: fair point. Didn't realise there was a `_THROW.SILENT`. have changed it

**Comment by samboylett** [comment]
I think I would prefer some kind of standard, e.g. `v-html` can only be used with a variable with a suffix of `_html`. I don't really want to disable the rule outright

**Comment by rowbi-wan** [comment]
> I think I would prefer some kind of standard, e.g. `v-html` can only be used with a variable with a suffix of `_html`. I don't really want to disable the rule outright

@samboylett The rule only allows turning on or off I'm afraid.

**Pull request merged**